my_list.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ ] Please make sure each method is implemented on your own with out the use of ruby's default "each" method as this project is intended for you to learn how enumerable methods work.
- [ ] **hint** : 
 **_Let us first understand how the default each method works :scientist:   _**
- it accepts a block as a parameter
- it iterates through each element of a list 
- for each element it executes what is with in the block passed
example: [1,2,3].each {|ele| puts ele * 2}
:- in the example above the each method iterates through each element of the array and executes `puts ele * 2` for each element

**_Now let us implement our own each method :muscle:_**
- it should expect to receive a block we can do the by using (&block) as a parameter which you already did.
- it should iterate through each element with out ruby's default "each" method, we can use a while loop and iterate for the length of the elements in the list.
example :
```
counter = 0
while counter < @list.length
       counter++
       
end
```